airbyte-commons/src/main/java/io/airbyte/commons/functional/VoidCallable.java

```diff
@@ -22,7 +22,7 @@
  * SOFTWARE.
  */
 
-package io.airbyte.commons.concurrency;
+package io.airbyte.commons.functional;
```

Comment:
@michel-tricot are you okay with me reusing and moving this to the functional package? i figure it was in concurrency just because it was originally only used by concurrency stuff.

Reply:
the original `Callable` is in the concurrency package because it is meant to be use with threadpools etc. 

The functional version of it is the `Supplier`.

Reply:
yeah. i guess the issue is that the thing i want to do is fundamentally not function. it takes nor arguments and has no return. it's pure side effect. going to just create an interface called procedure. i will restore the VoidCallable stuff to the way it was.